File: Samples/CS/07-RibbonColor/Form1.cs
using System;
using System.Drawing;
using System.Windows.Forms;
using RibbonLib;

namespace _07_RibbonColor
{
    public partial class Form1 : Form
    {

        public Form1()
        {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            // set ribbon colors
            _ribbon.SetColors(Color.Wheat, Color.IndianRed, Color.BlueViolet);
        }
    }
}

File: Samples/VB/07-RibbonColor/Form1.vb
Imports RibbonLib

Namespace _07_RibbonColor
	Partial Public Class Form1
		Inherits Form

		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			' set ribbon colors
			_ribbon.SetColors(Color.Wheat, Color.IndianRed, Color.BlueViolet)
		End Sub
	End Class
End Namespace
